stdlib/2and3/distutils/command/install.pyi

```diff
@@ -0,0 +1,14 @@
+from distutils.cmd import Command
+from typing import Union, Optional
+
+
+class install(Command):
+    user: Union[int, bool]
```

Comment:
While this field is initialized with `0` in the implementation (probably a relic from back when there were no booleans in Python), it is clearly intended to be a boolean. Just annotate with `bool`.